src/main.rs

```diff
@@ -45,8 +45,8 @@ fn opts() -> Opts {
     let image_format = short('f')
         .long("format")
         .help(
-            "image format of generated image. Only PNG or SVG is valid (case-insensitive). \
-            PNG will be used if not specified.",
+            "image format of generated image. Only PNG or SVG is valid (case-insensitive).
+PNG will be used if not specified.",
```

Comment:
Why is this change needed?

Reply:
It's not needed but it makes it so the help message fits into 120 columns wide screen rather than 150 one. I can drop it if you prefer.

Reply:
Makes sense, thanks for clarifying!

Reply:
While at it - I released 0.5.3, if you just bump the version users will get better output in case of a typo in a flag/command name. No other changes should be required.

Reply:
Well, regenerate a lock file if you ship one.

Reply:
Thanks for the tip! Updated it along with other deps: https://github.com/JohnTitor/ranpha/commit/ef2a49b0c2224462a788b5f3a95d9783ca92c3c2